app-admin/yadm/yadm-3.2.2.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..10} )
```

Comment:
Could you ping upstream about python-3.11 support? I tried and the tests fail, but don't see an upstream ticket about it. (Maybe it's fixed in -git already?)

Reply:
i will open a ticket with the upstream